i18n/validate.py

```diff
@@ -53,9 +54,9 @@ def msgfmt_check_po_file(locale_dir, filename):
     # Use relative paths to make output less noisy.
     rfile = os.path.relpath(filename, locale_dir)
     out, err = call('msgfmt -c -o /dev/null {}'.format(rfile), working_directory=locale_dir)
-    if err != '':
-        log.info('\n' + out)
-        log.warning('\n' + err)
+    if err:
+        log.info(six.u('\n') + out.decode('utf8'))
+        log.warning(six.u('\n') + err.decode('utf8'))
```

Comment:
I am not sure how I can know for certain what the stdout and stderr encoding is for this command. `utf8` seems to work in the cases I have tested. @nedbat, do you have any suggestions as to how we can be more confident about this?